File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        largest=float('-inf')
        for i in accounts:
            count=0
            for j in i:
                count+=j
            if count>largest:
                largest=count
        return largest

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int max=0;
        for(auto& i : accounts){
            int count=0;
            for(int j : i){
                count+=j;
            }
            if(count>max){
                max=count;
            }
        }
        return max;
    }
};